contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,109 @@
+.. _doc_ways_to_contribute:
+
+How to contribute
+=================
+
+The Godot Engine is free and open-source. Like any community-driven project, we rely on volunteer contributions. 
+On this page we want to showcase the various ways you as users can participate - to help you find the right starting place with the skillset you have.
+Because contrary to popular opinion, we need more than just programmers on the project!
+
+
+Fundraising
+-----------
+
+- **Donate**
+
+  We created the non-profit `Godot Foundation <https://godot.foundation/>` to be able to support the Godot Engine in both matters of finance and administration. 
+  In action, this means we can hire people to work part-time or full-time on the project. 
+  These jobs include engine development as well as related tasks like code reviews, production management, community & marketing efforts, and more.
+
+  With as little as 5 Euro per month, you can help us keep going strong. 
+  Currently, we are intending to hire more core developers, as to cover more ground with full-time specialists that suplement and guide volunteer work.
+
+  `Join the Development Fund <fund.godotengine.org>`
+
+- **Donation Drives**
+  Think about your followers on social media, or other communities you are active in.
+  Use that reach to remind your social environment that even small contributions can make a difference, especially when done by a great number of people at the same time. 
+
+  Are you a content creator? Consider adding a link to the `Godot Development Fund <fund.godotengine.org>` to your descriptions. 
+  If you do live streams, perhaps think about organizing a stream with donation incentives.
+
+.. - **Buy Official Merch**
+
+- **Publish Godot Games.**
+  You heard right, simply publishing a game #MadeWithGodot can positively impact the well-being of this project. 
+  Your personal success elevates the engine to a viable alternative for other developers, growing the community further. 
+  Additionally, it opens the doors for us to approach industry contacts aboout possible cooperations.
+
+
+Technical contributions
+-----------------------
+
+- **Report bugs & other issues**
+  As active users of the engine, you are better equipped to identify bugs and other issues than anyone else.
+  To let us know about your findings, fill out `this bug report form <https://github.com/godotengine/godot/issues/new?assignees=&labels=&projects=&template=bug_report.yml>` on our GitHub.
```

Comment:
```suggestion
  To let us know about your findings, fill out `this bug report form <https://github.com/godotengine/godot/issues/new?template=bug_report.yml>` on our GitHub.
```

Also maybe it would be better to link to `https://github.com/godotengine/godot/issues/new` so that the user can select the report category that best fits their needs.

Reply:
Good call! Will change this